globals.json

```diff
@@ -453,327 +347,69 @@
 		"onresize": true,
-		"onscroll": true,
-		"onsearch": true,
-		"onseeked": true,
-		"onseeking": true,
-		"onselect": true,
-		"onstalled": true,
-		"onstorage": true,
-		"onsubmit": true,
-		"onsuspend": true,
-		"ontimeupdate": true,
-		"ontoggle": true,
-		"ontransitionend": true,
-		"onunhandledrejection": true,
 		"onunload": true,
-		"onvolumechange": true,
-		"onwaiting": true,
-		"onwheel": true,
 		"open": false,
 		"openDatabase": false,
 		"opener": false,
 		"Option": false,
 		"origin": false,
-		"OscillatorNode": false,
-		"outerHeight": false,
-		"outerWidth": false,
-		"PageTransitionEvent": false,
-		"pageXOffset": false,
-		"pageYOffset": false,
-		"PannerNode": false,
 		"parent": false,
-		"Path2D": false,
-		"PaymentAddress": false,
-		"PaymentRequest": false,
-		"PaymentRequestUpdateEvent": false,
-		"PaymentResponse": false,
 		"performance": false,
-		"Performance": false,
-		"PerformanceEntry": false,
-		"PerformanceLongTaskTiming": false,
-		"PerformanceMark": false,
-		"PerformanceMeasure": false,
-		"PerformanceNavigation": false,
-		"PerformanceNavigationTiming": false,
-		"PerformanceObserver": false,
-		"PerformanceObserverEntryList": false,
-		"PerformancePaintTiming": false,
-		"PerformanceResourceTiming": false,
-		"PerformanceTiming": false,
-		"PeriodicWave": false,
-		"Permissions": false,
-		"PermissionStatus": false,
-		"personalbar": false,
-		"PhotoCapabilities": false,
-		"Plugin": false,
-		"PluginArray": false,
-		"PointerEvent": false,
-		"PopStateEvent": false,
-		"postMessage": false,
-		"Presentation": false,
-		"PresentationAvailability": false,
-		"PresentationConnection": false,
-		"PresentationConnectionAvailableEvent": false,
-		"PresentationConnectionCloseEvent": false,
-		"PresentationConnectionList": false,
-		"PresentationReceiver": false,
-		"PresentationRequest": false,
 		"print": false,
-		"ProcessingInstruction": false,
-		"ProgressEvent": false,
-		"PromiseRejectionEvent": false,
-		"prompt": false,
-		"PushManager": false,
-		"PushSubscription": false,
-		"PushSubscriptionOptions": false,
 		"queueMicrotask": false,
-		"RadioNodeList": false,
 		"Range": false,
-		"ReadableStream": false,
-		"registerProcessor": false,
-		"RemotePlayback": false,
 		"removeEventListener": false,
 		"Request": false,
 		"requestAnimationFrame": false,
-		"requestIdleCallback": false,
 		"resizeBy": false,
-		"ResizeObserver": false,
-		"ResizeObserverEntry": false,
 		"resizeTo": false,
 		"Response": false,
-		"RTCCertificate": false,
-		"RTCDataChannel": false,
-		"RTCDataChannelEvent": false,
-		"RTCDtlsTransport": false,
-		"RTCIceCandidate": false,
-		"RTCIceGatherer": false,
-		"RTCIceTransport": false,
-		"RTCPeerConnection": false,
-		"RTCPeerConnectionIceEvent": false,
-		"RTCRtpContributingSource": false,
-		"RTCRtpReceiver": false,
-		"RTCRtpSender": false,
-		"RTCSctpTransport": false,
-		"RTCSessionDescription": false,
-		"RTCStatsReport": false,
-		"RTCTrackEvent": false,
```

Comment:
@sindresorhus  something is missing. I guess we need to combine that `jshint` import method with the previous one somehow.